lib/improver/utilities/cube_checker.py

```diff
@@ -171,3 +171,49 @@ def find_dimension_coordinate_mismatch(
     else:
         mismatch = list(set(second_dim_names) - set(first_dim_names))
     return mismatch
+
+
+def coord_exists_in_cube(cube, coord_name):
+    """
+    Test whether a coordinate exists within a cube, regardless of whether it
+    is a dimension, auxillary or scalar coordinate.
+
+    Args:
+        cube : iris.cube.Cube
+            The cube to be tested.
+        coord_name : str
+            The name of the coordinate to be looked for.
+
+    Returns:
+        bool
+            True if the coordinate is found within the cube, False if it is
+            not.
+
+    """
+
+    if [coord.name() for coord in cube.coords()
+            if coord_name in coord.name()]:
+        return True
+    return False
+
+
+def coord_is_dimension(cube, coord_name):
+    """
```

Comment:
Where are the unit tests for this function

Reply:
This has been removed as no longer used